.teamcity/src/templates/KibanaTemplate.kt

```diff
@@ -0,0 +1,129 @@
+package templates
+
+import vcs.Kibana
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.ParameterDisplay
+import jetbrains.buildServer.configs.kotlin.v2019_2.Template
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.placeholder
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+
+object KibanaTemplate : Template({
+  name = "Kibana Template"
+  description = "For builds that need to check out kibana and execute against the repo using node"
+
+  vcs {
+    root(Kibana)
+
+    checkoutDir = "kibana"
+//    checkoutDir = "/dev/shm/%system.teamcity.buildType.id%/%system.build.number%/kibana"
```

Comment:
Was this swapped for testing?

Reply:
`"/dev/shm/%system.teamcity.buildType.id%/%system.build.number%/kibana"` puts the workspace for the job in memory, but I had some issues when I tried it initially. It's on my TODO list to revisit. It has pretty good performance without being in-memory (probably since a ton of stuff isn't all running on one machine), so it's not a huge concern at the moment.